build-static.sh

```diff
@@ -187,14 +187,16 @@ export CGO_CPPFLAGS="${CGO_CFLAGS}"
 if [ "${os}" = "mac" ]; then
 	export CGO_LDFLAGS="-framework CoreFoundation -framework SystemConfiguration"
 elif [ "${os}" = "linux" ] && [ -z "${DEBUG_SYMBOLS}" ]; then
-	CGO_LDFLAGS="-Wl,-O1 -pie"
+	CGO_LDFLAGS="-Wl,-O1 -pie -Wl,--allow-multiple-definition -Wl,--export-dynamic"
```

Comment:
--export-dynamic is necessary to prevent it from stripping symbols out even if --whole-archive is used.

--allow-multiple-definition is necessary with the --whole-archive for libphp.a